aggsender/types/interfaces.go

```diff
@@ -159,3 +171,12 @@ type MaxL2BlockNumberLimiterInterface interface {
 	AdaptCertificate(
 		buildParams *CertificateBuildParams) (*CertificateBuildParams, error)
 }
+
+type VerifyIncommingRequests struct {
```

Comment:
nitpick, typo?
```suggestion
type VerifyIncomingRequests struct {
```

As a side note, does it make sense to name it as singular `VerifyIncomingRequest`?

Reply:
I agree to both!